src/components/Header/Header.css

```diff
@@ -0,0 +1,44 @@
+.bg-black {
+    background-color: #000;
+}
+nav a {
+    display: inline-block;
+}
+
+nav a:hover{
+    background-color: #fff;
+    color:#000;
+    border-radius: 0;
+    cursor: pointer;
+}
+
+.menuButton {
+    display: none;
+}
+
+.inputButton {
+    display: none;
+}
+
+/* Mobile */
+
+@media screen and (max-width: 400px) {
+
+    nav {
```

Comment:
This will target all `<nav>` elements in the page. We should only target the nav inside this header.

I suggest you:

1. Add a class on the `<header>`, I suggest you name it the same as the component: `Header`.
2. Change all rules here to only target elements inside `<header>`. For example this one should be `.Header nav`

Reply:
Make sure you do this for all rules here.

Reply:
You didn't do this for all rules. For example, you now have a `logo` class here. If on a page we'll have other logos from different brands, also with the class `.logo` the rule will apply to them as well.